native/cocos/platform/empty/modules/SystemWindow.cpp

```diff
@@ -39,7 +39,7 @@ namespace {
 } // namespace
 
 namespace cc {
-SystemWindow::SystemWindow() {
+SystemWindow::SystemWindow(IEventDispatch* delegate) {
```

Comment:
Why modify it?

Reply:
It didn't use this interface with EmptyPlatform before, but WindowsPlatform used this interface.

Reply:
I did't find the codes to use it.